client/packages/common/src/ui/components/modals/ConfirmationModal/useConfirmationModal.ts

```diff
@@ -8,6 +8,8 @@ import {
 export const useConfirmationModal = ({
   onConfirm,
   message,
+  info,
+  buttonLabel = 'OK',
```

Comment:
buttonLabel should always be translated... might be easier not to provide default here, and if buttonLabel is undefined, can do the `t('label.ok')` in the ConfirmationModal component?

Reply:
Just missing this bit?

Reply:
😅 Thanks, this is updated now!